ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml

```diff
@@ -2064,6 +2064,30 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-gcp-upi-private-xpn-sdn
+- as: ocp-e2e-ibmcloud-ovn-multi-ppc64le-cucushift
+  cron: 20 23 * * *
+  steps:
+    cluster_profile: ibmcloud-multi-ppc64le
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      ADDITIONAL_WORKER_ARCHITECTURE: ppc64le
+      BASE_DOMAIN: ocp-dev-ppc64le.com
+      E2E_RUN_TAGS: '@ppc64le and @ibmcloud-ipi and @heterogeneous and @network-ovnkubernetes
+        and not @fips'
+      E2E_SKIP_TAGS: not @upgrade not @customer and not @destructive and not @flaky
```

Comment:
possible encoding issue with the tags? `'`

Reply:
Updated the code with `'` for E2E_SKIP_TAGS.